### PR TITLE
Feat/eia 459 eApp steps page

### PR DESCRIPTION
**Pull request by RichardBray**
# Description

https://consular.atlassian.net/browse/EIA-459

The purpose of this PR is to add the steps page below to the eApp flow of the page

Screenshot 2022-04-20 at 15 46 12

---

This page is to give users an idea of what steps they need to take in order to complete the eApp.

Feel free to run through [this prototype](https://fcdo-e-apostille-prototype.herokuapp.com/application/) to understand how this page fits in.

### CODE REVIEW

File: api/controllers/EAppEligibilityQuestionsController.js [hunk 1]
```diff
@@ -5,21 +5,34 @@ const EAppEligibilityQuestionsController = {
...
+            const signInUrl = `${req._sails.config.customURLs.userServiceURL}/sign-in?from=start&next=continueEApp`;
+            const skipLink = loggedIn ? '/upload-files' : signInUrl;
+
+            skipLinkObj = { skipLink };
+        }
 
         return res.view(questionPage, {
             user_data: HelperService.getUserData(req, res),
             page_error: false,
+            ...skipLinkObj,
```
Comment (PaulioRandall) [inline api/controllers/EAppEligibilityQuestionsController.js]: Just a little confused. The `skipLinkObj` is initialised as an array but assigned an object. Mind my ignorance of JS but might that not cause an issue when used?

**Review by PaulioRandall — COMMENTED** [review]
Looks good. Might be worth a quick walkthrough if you have time so I get more context on the project.

File: api/controllers/EAppEligibilityQuestionsController.js (hunk 1 above)
Comment (RichardBray) [reply api/controllers/EAppEligibilityQuestionsController.js]: Good spot. I've having some issue with dependencies at the movement but I'll take a look at it when  I've got that fixed.

**Review by PaulioRandall — APPROVED** [review]
LGTM

**Pull request merged**